code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -21,7 +21,7 @@
 		tally -= 0.5
 
 	var/health_deficiency = (maxHealth - health)
-	var/hunger_deficiency = (max_nutrition - nutrition) //400 = max for humans.
+	var/hunger_deficiency = (400 - nutrition)
```

Comment:
I'm sure there is a lot more code that uses `max_nutrition`. you should replace it with a proc or a macro instead of a number.

Reply:
`max_nutrition` has suprisingly little uses in the code. Humans inherit base nutrition all the way from the base `/mob` type, so i changed it to a define.